squidpy/gr/_build.py

```diff
@@ -95,6 +95,11 @@ def spatial_neighbors(
 
     transform = Transform.NONE if transform is None else Transform(transform)
     if coord_type is None:
+        if radius is not None:
+            warnings.warn(
```

Comment:
Please use `logg.warning` so that it's controlled via `scanpy.verbosity`

Reply:
Done